custom/active-speaker/components/SpeakerView/SpeakerTile/SpeakerTile.js

```diff
@@ -49,13 +49,14 @@ export const SpeakerTile = ({ participant, screenRef }) => {
   const style = {
     height,
     maxWidth: screenHeight * finalRatio,
+    outline: `1px solid ${defaultTheme.background}`,
     overflow: 'hidden',
   };
 
   return (
     <Tile
       aspectRatio={finalRatio}
-      participant={participant}
+      sessionId={sessionId}
```

Comment:
are these changes only converting to daily hooks or are there additional changes?